src/main/scala/nl/knaw/dans/easy/transform/XmlTransformation.scala

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (C) 2019 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.transform
+
+import java.net.URI
+
+import nl.knaw.dans.easy.transform.AccessRights.AccessRights
+
+import scala.xml.transform.{ RewriteRule, RuleTransformer }
+import scala.xml.{ Elem, Node }
+
+object XmlTransformation {
+
+  private val accessibleToRightsMap = Map(
```

Comment:
@lindareijnhoudt are these the only three values we want to support here? As @janvanmansum discovered in [EASY=2407](https://drivenbydata.atlassian.net/browse/EASY-2407?focusedCommentId=41107&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-41107), there are more values present in the vault.

Reply:
Added mapping `AccessRights.OPEN_ACCESS_FOR_REGISTERED_USERS -> AccessibleToRights.KNOWN`